ExampleGallery/Effects/VariableVideoEffect.cs

```diff
@@ -0,0 +1,84 @@
+﻿using Windows.Foundation.Collections;
+using Windows.Graphics.DirectX.Direct3D11;
+using Windows.Media.Effects;
+using Windows.Media.MediaProperties;
+using Microsoft.Graphics.Canvas;
+using Microsoft.Graphics.Canvas.Effects;
+using System.Collections.Generic;
+
+namespace ExampleGallery.Effects
+{
+    /// <summary>
+    /// Win2D Emboss Effect http://microsoft.github.io/Win2D/html/T_Microsoft_Graphics_Canvas_Effects_EmbossEffect.htm
+    /// Amount - min is 0f, max is 10f, default is 1f
+    /// </summary>
+    public sealed class VariableVideoEffect : IBasicVideoEffect
+    {
+        private CanvasDevice _canvasDevice;
+        private IPropertySet _configuration;
+
+        /// <summary>
+        /// The intensity of the Emboss Effect
+        /// </summary>
+        public float Amount
+        {
+            get
+            {
+                //check to see if there is an "Amount" key and get the value
+                if (_configuration != null && _configuration.ContainsKey("Amount"))
+                    return (float)_configuration["Amount"];
```

Comment:
Crashes here because the property set is initialized containing a double, not float value